src/gui/widgets/LcdFloatSpinBox.cpp

```diff
@@ -129,6 +130,10 @@ void LcdFloatSpinBox::contextMenuEvent(QContextMenuEvent* event)
 
 void LcdFloatSpinBox::mousePressEvent(QMouseEvent* event)
 {
+	// switch between integer and fractional step based on cursor position
+	if (event->x() < m_wholeDisplay.width()) { m_intStep = true; }
+	else { m_intStep = false; }
```

Comment:
```suggestion
	m_intStep = event->x() < m_wholeDisplay.width();
```
This is simpler

Reply:
Yeah, I just moved the code from one spot to another from the original code and didn't bother revising it.  Fixed now.